lexibank_tls.py

```diff
@@ -617,36 +615,58 @@
 ]
 
 
-class Dataset(BaseDataset):
+@attr.s
+class CustomConcept(pylexibank.Concept):
+    Swahili_Gloss = attr.ib(default=None)
+    Source_ID = attr.ib(default=None)
```

Comment:
CAn you change this into NUMBER, as we have the number here, right? This is a concepticon "standard" field.